lib/inspec/utils/telemetry/http.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require_relative "base"
+require "faraday" unless defined?(Faraday)
+module Inspec
+  class Telemetry
+    class HTTP < Base
+      TELEMETRY_JOBS_PATH = "v1/job"
+      TELEMETRY_URL = ENV["TELEMETRY_URL"] # TODO Set to production telemetry URL before release
```

Comment:
Does this need to be updated to CHEF_TELEMETRY_URL?

Reply:
Yes nice catch. I guess this went away because of a rebase https://github.com/inspec/inspec/commit/802d261642f041991d7d519e46a97b176e7df525

Let me revert it. Thanks